kvstore/kvstore_server.h

```diff
@@ -9,14 +10,16 @@ using kvstore::PutRequest;
 using kvstore::PutReply;
 using kvstore::GetRequest;
 using kvstore::GetReply;
+using kvstore::RemoveRequest;
+using kvstore::RemoveReply;
 
 class KvstoreServiceImpl final : public KeyValueStore::Service {
 public:
   Status put(ServerContext* context, const PutRequest* request, PutReply* reply){}
   Status get(ServerContext* context, const GetRequest* request, GetReply* reply){}
   Status remove(ServerContext* context, const RemoveRequest* request, RemoveReply* reply){}
 private:
-  std::unordered_map map_;
+  std::unordered_map<std::string, std::string> map_;
```

Comment:
refer to above comments about abstraction of this kvstore

Reply:
did i do it right